spec/models/article_spec.rb

```diff
@@ -29,4 +29,39 @@
       end
     end
   end
+
+  context 'scopes' do
+    describe 'visible' do
+      it 'should respond' do
+        expect(Article).to respond_to :visible
+      end
+      it 'should return all articles with visible true' do
+        visible = FactoryGirl.create_list :article, 5, visible: true
+        FactoryGirl.create_list :article, 5, visible: false
+        expect(Article.visible).to match_array visible
+      end
+    end
+
+    describe 'available' do
+      it 'should respond' do
+        expect(Article).to respond_to :available
+      end
+      it 'should return articles with date < the now' do
+        available = FactoryGirl.create_list :article, 5, date: Time.now - 5.days
+        FactoryGirl.create_list :article, 5, date: Time.now + 5.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.